stage #2.5/module #4 REST Basics - Spring Web/SpringWeb/content.md

```diff
@@ -0,0 +1,30 @@
+## Spring Web
+
+### What is Spring Web?
```

Comment:
Looks like this chanpter is not required at all.
Here you provide links to the resources you've already described in previous chapters